src/backend/joanie/core/api/admin.py

```diff
@@ -111,6 +114,19 @@ def get_queryset(self):
 
         return super().get_queryset()
 
+    @action(
+        detail=False,
+        methods=["get"],
+        permission_classes=[permissions.IsAuthenticated],
```

Comment:
why not : permissions.IsAdminUser & permissions.DjangoModelPermissions
for coherence with the other endpoints of this admin api?